text_volpiano_alignment.py

```diff
@@ -101,8 +101,12 @@ def _align_word(text_word: "list[str]", volpiano_word: str) -> "list[tuple[str,
     # syllables in the text than in the volpiano.
     if len(text_word) > len(vol_syls):
         squashed_txt_word = text_word[: len(vol_syls) - 1]
-        squashed_txt_word.append("".join(text_word[len(vol_syls) - 1 :]))
+        squashed_final_syl = "".join(text_word[len(vol_syls) - 1 :])
+        squashed_txt_word.append(squashed_final_syl)
```

Comment:
wait... what exactly does "squashed" mean here? I like the look of the screenshots in this PR's initial comment, but looking at this change, I don't understand what's going on.

Reply:
...wait - this is for the situation where there are three volpiano syllables for "al-le-lu-ia" - it becomes `["al-", "le-", "lu-ia"], right?

Reply:
yes...so "squashed" here (oof, not very informative name is right!) means that if you have `m` volpiano syllables and `m + n` text syllables, text syllables `m` through `m+n` are all joined together as the "final" text syllable and aligned with the `n`th volpiano syllable.

Reply:
how about `overhanging_text_syls` or `additional_text_syls`?

Reply:
`additional` sounds more proper/literal, but `overhanging` feels more descriptive/evocative. Go with `overhanging_[text|volpiano]_syls`!

Reply:
I think variable names should now be clearer... let me know what you think!